python/lsst/obs/monocam/monocamMapper.py

```diff
@@ -78,7 +78,7 @@ def _computeCcdExposureId(self, dataId):
         @param dataId (dict) Data identifier with visit
         """
         visit = dataId['visit']
-        return long(visit)
+        return int(visit)
```

Comment:
We have a long standing issue as to whether we care about `long` on Python 2. There is no difference on 64-bit python 2 because `int` is 64-bit and that's what `long` is. On 32-bit we need to explicitly use `long` but given that we no longer support 32-bit python 2 I think dropping of `long` completely is fine.